runtime/queries/typescript/textobjects.scm

```diff
@@ -0,0 +1,7 @@
+; inherits: javascript
+
+(interface_declaration
+  name: (_) ) @class.around
+  
+(type_alias_declaration
+  name: (_) ) @class.around
```

Comment:
we don't use the name node here so the pattern match can be removed:

```
(interface_declaration) @class.around
(type_alias_declaration) @class.around
```

Or they could be combined

```
[
  (interface_description)
  (type_alias_declaration)
] @class.around
```

Reply:
have change like this 
```
[
  (interface_declaration 
    body:(_) @class.inside)
  (type_alias_declaration 
    value: (_) @class.inside)
] @class.around
```